docs/update-curl-utility.md

```diff
@@ -1,42 +1,106 @@
-# Update the curl utility in Debian 10
+# Dependency compatibility and limitations
 
-The default curl version, 7.64.0, in Debian 10 has known issues when
-attempting to reuse an already closed connection. This issue directly
-affects `xbcloud` and users may see intermittent backup failures.
+`xbcloud` depends on several system libraries for correct operation. Some versions of these dependencies are known to cause unexpected terminations or
+unstable behavior and are therefore not supported.
 
-For more details,
-see [curl #3750](https://github.com/curl/curl/issues/3750)
-or [curl #3763](https://github.com/curl/curl/pull/3763).
+If you are experiencing unexpected terminations or intermittent failures, verify that your environment does not use any of the versions listed below.
 
-Follow these steps to upgrade curl to version 7.74.0:
+## libcurl
+
+Certain versions of `libcurl` are known to cause unexpected terminations or connection
+reuse issues in `xbcloud`. The following versions are not supported.
+
+### Unsupported versions
+
+`libcurl 7.64.0 `
+
+* Known to mishandle reuse of closed connections
+
+* May cause intermittent backup failures in `xbcloud`
+
+`libcurl from 8.11.1 up to, but not including, 8.12.0`
+
+* Affected by an event loop shutdown issue, [CVE-2025-0665](https://github.com/advisories/GHSA-cc57-hgv8-p56r)
+
+* May cause unexpected terminations when `xbcloud` uses `libcurl` with `libev`
+
+### Upgrade or downgrade options
+
+* Upgrade to libcurl `8.12.0 or later` if these versions are compatible with your operating system.
```

Comment:
Should have the link to the upgrade or downgrade instructions.

Reply:
I've provided the instructions further down in the document.